stdlib/http/src/main/ballerina/http/websocket/websocket_connector.bal

```diff
@@ -78,11 +79,11 @@ type WebSocketConnector object {
     #                   until a close frame is received. If WebSocket frame is received from the remote endpoint,
     #                   within waiting period the connection is terminated immediately.
     # + return - `error` if an error occurs when sending
-    public function close(int? statusCode = 1000, string? reason = (), int timeoutInSecs = 60) returns error? {
+    public function close(int? statusCode = 1000, string? reason = (), int timeoutInSecs = 60) returns WebSocketError? {
         if (statusCode is int) {
             if (statusCode <= 999 || statusCode >= 1004 && statusCode <= 1006 || statusCode >= 1012 &&
                 statusCode <= 2999 || statusCode > 4999) {
-                error err = error("Failed to execute close. Invalid status code: " + statusCode);
+                WebSocketError err = error(message = "Failed to execute close. Invalid status code: " + statusCode);
```

Comment:
You cannot do this. The `WebsocketError` cannot be crated directly, it is a union type.